### ISSUE TITLE
Energy Ball redirection cube

### ISSUE THREAD

**Issue by Milkman68** [ENDERZOMBI102/Retro-Station#14]
This is from the map Energy Redirection https://steamcommunity.com/sharedfiles/filedetails/?id=514318341 I’m away from my Computer so I can’t provide you with a Zip but it would be a really great Addition for Pellet Puzzle’s

**Comment by Flash4433** [comment]
There's no need for zips, I can decompile it myself just so you don't need to upload the files. I think this item shouldn't be used at all, since it's basically a reflection cube that's easily exploitable. But well, it's a suggestion.

**Comment by ENDERZOMBI102** [comment]
this can already be replicated with a normal cube.
pellets are refleclet by them

**Comment by Flash4433** [comment]
Yeah, and that can be enabled/disabled in the ItemVars if I'm not wrong. However, I'm pretty sure this cube works like a reflection cube except it makes energy balls come out of the front of the cube, like lasers. Kinda hard to explain...
